crates/fuzzing/src/generators/table_ops.rs

```diff
@@ -85,29 +108,20 @@ pub(crate) enum TableOp {
 }
 
 impl TableOp {
-    fn to_wat_string(&self, wat: &mut String) {
+    fn to_instruction(&self) -> Instruction {
```

Comment:
One option here is that instead of `to_instruction` this could be something like `insert(&self, func: &mut Function)` and that way you can insert multiple functions if necessary.

Reply:
This is a great idea -- I took a stab at it!